express_relay/sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -59,13 +60,24 @@ class SimpleSearcherLimo {
     );
   }
 
+  async getMintDecimals(mint: PublicKey): Promise<number> {
+    if (this.mintDecimals[mint.toBase58()]) {
+      return this.mintDecimals[mint.toBase58()];
+    }
+    const decimals = await getMintDecimals(this.connectionSvm, mint);
```

Comment:
nit, using the same name for this method is a bit confusing

Reply:
maybe `mintDecimals`?

Reply:
added a cached prefix